ring-jetty-adapter/src/ring/adapter/jetty.clj

```diff
@@ -96,8 +97,17 @@
       (.setIdleTimeout (options :max-idle-time 200000)))))
 
 (defn- ^ThreadPool create-threadpool [options]
-  (let [pool (QueuedThreadPool. ^Integer (options :max-threads 50))]
-    (.setMinThreads pool (options :min-threads 8))
+  (let [min-threads (options :min-threads 8)
+        queue-capacity (max min-threads 8)
+        queue-max-capacity (options :max-queued-requests Integer/MAX_VALUE)
+        blocking-queue (BlockingArrayQueue. ^Integer queue-capacity
+                                            ^Integer queue-capacity
+                                            ^Integer queue-max-capacity)
+        thread-idle-timeout (options :thread-idle-timeout 60000)
+        pool (QueuedThreadPool. ^Integer (options :max-threads 50)
```

Comment:
Lets put `max-threads` in the `let` clause as well, so that it's in the same place as `min-threads`. I think that will make the code a little more readable.